tests/continuous/test_nse.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np
```

Comment:
Can you also add tests that check that the behaviour is correct, if the denominator is zero?

Reply:
Can you add tests that check that the `preserve_dims` and `reduce_dims` work as expected?